e2e_samples/parking_sensors_synapse/devops/azure-pipelines-ci-artifacts.yml

```diff
@@ -68,7 +68,28 @@ stages:
         PathtoPublish: '$(basePath)/synapse'
         ArtifactName: 'synapse'
       displayName: 'Publish Synapse Artifacts'
+    
+    - task: PublishBuildArtifacts@1
```

Comment:
Since this is running the parsingSQLJson job, might be worth put these three tasks under its own job. So basically create a new job like 'extract_and_publish_sql_serverless_sql' and move tasks to that job.